data/mode.txt

```diff
@@ -1 +1 @@
-sandbox
+Dynamic Mode
```

Comment:
:thinking:

Reply:
dangert, why didn't you let me fix this before you merged it?